arlo-client/src/components/Audit/useContests.ts

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react'
```

Comment:
moved from `useContestsApi`, but git didn't pick up the change because there were also some small changes (e.g. adding `refreshId`)